docs/vision-usage.rst

```diff
@@ -283,6 +283,38 @@ image and determine the dominant colors in the image.
     0.758658
 
 
+*********************
+Batch image detection
+*********************
+
+Multiple images can be processed with a single request by passing ``Image``s to
+``Client.batch(images)``.
```

Comment:
You can / should link to the actual `Image` class and the actual `Client.batch` method